src/main/java/de/szut/lf8_project/project/ProjectService.java

```diff
@@ -36,5 +36,16 @@ public ProjectEntity readById(long id) {
     public List<ProjectEntity> readAll() {
         return this.repo.findAll();
     }
+
+    public void delete(long id) {
```

Comment:
deleteById would be nicer i guess since we are not directly deleting a present projectentity object